src/odemis/gui/cont/acquisition/snapshot.py

```diff
@@ -133,14 +134,16 @@ def _get_snapshot_info(self, dialog=False):
 
         return tab, filepath, exporter
 
-    def snapshot_viewport(self, tab, filepath, exporter, anim):
+    def snapshot_viewport(self, tab, filepath, exporter, anim, forcemd: Optional[dict] = None):
```

Comment:
Please adjust the plugin such that it doesn't need to add this option to the standard GUI code. This option is very unlikely going to be used by the standard code. In addition, it should be possible to find other way to do the same behaviour directly in the plugin. I think the best would be to let the plugin directly export the .raw data. Alternatively, it could update the .metadata by itself. No need to do it here.